packages/lesswrong/components/tagging/TagNotificationSettings.tsx

```diff
@@ -99,6 +106,22 @@ const SubforumNotificationSettings = ({
     fragmentName: 'UserTagRelDetails',
   });
 
+  if (!isSubforum) {
+    return (
+      <AnalyticsContext pageSection="tagNotificationSettings">
+        <NotifyMeButton
+          document={tag}
+          tooltip={`Click to toggle notifications for ${taggedPostWording}`}
+          showIcon
+          hideLabel
+          hideIfNotificationsDisabled={!isFrontpageSubscribed}
+          subscriptionType={subscriptionTypes.newTagPosts}
+          className={classes.nonSubforumButtonWrapper}
+        />
+      </AnalyticsContext>
```

Comment:
For normal tag pages this essentially passes straight through to the `NotifyMeButton` which was previously in `SubscribeButton`